railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -547,6 +547,28 @@ def run_after_bundle_callbacks
         @after_bundle_callbacks.each(&:call)
       end
 
+      def show_next_steps_banner
+        return if options[:quiet]
+
+        puts <<~NEXTSTEP
+
+          Rails project created and dependencies installed.
+
+          Now switch to the directory where the Rails app was created:
+
+            $ cd #{app_path}
+
+          Then boot the development server:
+
+            $ ./bin/rails server
+
+          You can always get help by running:
+
+            $ ./bin/rails help
+
+        NEXTSTEP
```

Comment:
Maybe we should add a link here to the getting started guide, specifically in the section after how to generate the app?

Reply:
To keep it brief, somewhat generic, and robust against the URL changing, I added this blurb:

```
More help is available in the guides at https://guides.rubyonrails.org/
```

I left out the API docs and a lot of other links because they can figure that out when they head to Guides and I don't want to overwhelm with a big list of links.